src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -54,6 +54,7 @@ object CommentStyleMapping {
 
 object HeaderKey {
   val headers = settingKey[Map[String, (Regex, String)]]("Header pattern and text by extension; empty by default")
+  val excludes = settingKey[Seq[String]]("File patterns for files to be excluded; empty by default")
```

Comment:
Is it a good idea to use `Seq[String]` here? Because of the redefinition of `Seq` to `scala.collection.immutable.Seq` in `package.scala`, users will have to fully qualify `scala.collection.immutable.Seq` in their build.sbt, since by default they will get a mutable Seq. See test.sbt in the exclude sbt-test project for an example of what this would look like.